libs/payments/ui/src/lib/client/components/Header/en.tfl

```diff
@@ -0,0 +1,30 @@
+# Component - Header
+
+next-header-help
```

Comment:
nit. Now that all l10n pipeline work has been completed, the `next-` prefix is no longer necessary.

Could the strings be updated to not include the `next-` prefix?

Reply:
Sure! Should I make new string ids entirely, or still reuse the ids from the fax settings components?

Reply:
New id's entirely please.

The only reason we reused id's and prefixed them with `next-` was that we could provide an initial set of available translations. However, that portion of work has been completed, and going forward we'll be creating only new strings.